pyproject.toml

```diff
@@ -40,14 +40,25 @@ sphinx-rtd-theme = "^0.4.3"
 
 
 [tool.poetry.dev-dependencies]
-pytest = "*"
-pylama = "*"
-pylint = "*"
-pylint_runner = "*"
+black = "^19.10b0"
+pytest = "^5.0"
+pylint = "^2.3"
+pylint_runner = "^0.5.4"
 sphinx = "^2.2.0"
 
+[tool.black]
+exclude = '''
+(
+     /(
+         docs
+     )/
+     | setup.py
+)
+'''
+
 [tool.dephell.main]
-from = {format = "poetry", path = "pyproject.toml"}
+from ={format="poetry", path="pyproject.toml"}
```

Comment:
I'm a bit puzzled about why the formatting in this line changes, but the formatting in the `to =` line below remained the same

Reply:
I set `isort` to auto run after save. For some reason, it insists woking on `toml` file :(